blocks/components/editable/index.js

```diff
@@ -329,6 +329,7 @@ export default class Editable extends wp.element.Component {
 		} else {
 			this.editor.formatter.apply( format );
 		}
+		this.editor.setDirty( true );
```

Comment:
Isn't this done automatically by the `formatter` calls? Sometimes calling a formatter doesn't change the content (end of text for example) and we may want to avoid this.